census/models/utils.js

```diff
@@ -133,13 +133,17 @@ var processEntries = function(data, options) {
     data.submitters = [];
 
     if (options.cascade) { data.entries = cascadeEntries(data.entries, options.year); }
+
+    _.each(data.entries, function(e) {
+      e.computedYCount = e.yCount(data.questions);
```

Comment:
Yes, I know. Can you check though? I think we only need it on entries, and therefore calculating it for all variants is a waste (but, I could be wrong, so please confirm)